conftest.py

```
@@ -281,19 +281,51 @@ def _add_upgrade_test(_item: Item, _upgrade_deployment_modes: list[str]) -> bool
+    enable_console_value = True  # Default to console enabled
+
+    # Check for explicit -o log_cli option in command line overrides
+    try:
+        overrides = getattr(session.config.option, "override_ini", []) or []
+        log_cli_override = next((override for override in overrides if override.startswith("log_cli=")), None)
+
+        if log_cli_override:
+            value = log_cli_override.split("=", 1)[1].lower()
+            enable_console_value = value not in ("false", "0", "no", "off")
+
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)